frontend/src/components/auth/Login.tsx

```diff
@@ -25,22 +25,31 @@ const Login = (): React.ReactElement => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [loginError, setLoginError] = useState(false);
+  const [invalidFormError, setinvalidFormError] = useState(false);
```

Comment:
```suggestion
  const [invalidFormError, setInvalidFormError] = useState(false);
```

Reply:
^ i don't think the change was made

Reply:
i made the loginerror that says "please fill in fields" and renamed its variables to invalidformerror and setinvalidform error and then used the loginerror and setloginerror variables for the "failed to login error" so I renamed the variables! is that what you were asking?

Reply:
ohh i meant to capitalize `setinvalidFormError` -> `setInvalidFormError`